lib/pbench/test/unit/server/test_datasets_inventory.py

```diff
@@ -76,13 +79,11 @@ def test_unauthorized_access(self, query_get_as):
 
     @pytest.mark.parametrize("key", (None, "", "subdir1"))
     def test_path_is_directory(self, query_get_as, monkeypatch, key):
-        def mock_extract(self, target):
-            info = {"type": CacheType.DIRECTORY}
-            stream = None
-            return (info, stream)
-
+        filestream_tuple = {"type": CacheType.DIRECTORY, "stream": None}
```

Comment:
`filestream_tuple` looks like a `dict`, not a "tuple", to me.  🙂 

(Ditto line 96.)